### PR TITLE
Update messages of the block sync extension

### PR DESCRIPTION
**Pull request by Remagpie**
Collected from https://github.com/CodeChain-io/codechain/pull/1843 and https://github.com/CodeChain-io/codechain/pull/1864

### CODE REVIEW

File: spec/Block-Synchronization-Extension.md [hunk 1]
```diff
@@ -19,13 +19,14 @@ Message :=
 ### Status
 
 ```
-Status(total_score, best_hash, genesis_hash)
+Status(nonce, best_hash, genesis_hash)
```
Comment (sgkim126) [inline spec/Block-Synchronization-Extension.md]: How about `seq`?
`nonce` seems a random number not a monotonically increasing number.

Comment (Remagpie) [reply spec/Block-Synchronization-Extension.md]: It seems better. Let me fix it.

**Comment by sgkim126** [comment]
Please describe what has been changed in https://github.com/CodeChain-io/foundry/pull/28/commits/3fe4b23c95d7274e11013ca8226f5b845db4020d and why do you change it.

**Comment by Remagpie** [comment]
I added a link to the research site to the comment.

**Pull request merged**